src/ui/Windows/PasswordSafe.rc

```diff
@@ -3267,7 +3268,8 @@ LANGUAGE 9, 1
 #include "..\..\core\core.rc2"    // non-MS VC++ edited core STRING resources
 #include "afxres.rc"         // Standard components
 #endif
-#endif

+#endif
+

```

Comment:
Since I didn't make any changes to this line, I don't understand why this shows up as changed content.

Reply:
Maybe your editor does some auto-formatting?

Reply:
I did this with two different editors, neither of which have auto formatting enabled. It's really strange why these two lines are affected. I assume it's an interpretation issue on the part of 'git diff'.

Reply:
Perhaps you have `core.autocrlf = true` in your .gitconfig? FWIW, mine has `core.autocrlf = input`.